Common/src/main/resources/assets/crafttweaker/lang/it_it.json

```diff
@@ -0,0 +1,103 @@
+  "crafttweaker.command.description.recipes.hand" : "Restituisce informazioni su tutte le ricette per l'oggetto nella mano",
+  "crafttweaker.command.description.recipes.inventory" : "Restituisce informazioni su tutte le ricette per gli oggetti nell'inventario del giocatore",
+  "crafttweaker.command.description.recipes.manager" : "Restituisce informazioni sulle ricette per il gestore di ricette selezionato",
+  "crafttweaker.command.description.reload" : "Reindirizza al comando /reload",
+  "crafttweaker.command.description.script" : "Apre la cartella degli script",
+  "crafttweaker.command.description.syntax" : "Controlla la sintassi degli script",
+  "crafttweaker.command.dump.generated" : "È stato generato un elenco delle espressioni tra parentesi uncinate di tipo '%s'!",
+  "crafttweaker.command.dump.types" : "Salva tipi",
+  "crafttweaker.command.example.generated" : "Esempi salvati nella cartella 'examples' allinterno della cartella degli script. Fai clic per aprire la cartella!",
+  "crafttweaker.command.files.created" : "I file sono stati creati. Fai clic per aprire la cartella!",
+  "crafttweaker.command.hand.header.attributes" : "Attributi di",
+  "crafttweaker.command.hand.header.tags.block" : "Tag di blocchi",
+  "crafttweaker.command.hand.header.tags.item" : " Tag di oggetti",
+  "crafttweaker.command.hand.no.data" : "L'oggetto non contiene dati",
+  "crafttweaker.command.hand.no.tags" : "L'oggetto non ha tag",
+  "crafttweaker.command.help.page.info" : "Pagina %s di %s",
+  "crafttweaker.command.info.block.activated" : "La modalità di analisi dei blocchi è stata attivata. Fai clic con il tasto destro su un blocco per vedere i suoi dati.",
+  "crafttweaker.command.info.block.deactivated" : "La modalità di analisi dei blocchi è stata disattivata",
+  "crafttweaker.command.info.block.entity.data" : "Dati dell'entità-blocco: %s",
+  "crafttweaker.command.info.block.name" : "Nome del blocco: %s",
+  "crafttweaker.command.info.block.properties" : "Proprietà: ",
+  "crafttweaker.command.info.entity.activated" : "La modalità di analisi delle entità è stata attivata. Fai clic con il tasto destro su un'entità per vedere i suoi dati.",
+  "crafttweaker.command.info.entity.data" : "Dati dell'entità: %s",
+  "crafttweaker.command.info.entity.deactivated" : "La modalità di analisi delle entità è stata disattivata",
+  "crafttweaker.command.info.entity.name" : "Nome dell'entità: %s",
+  "crafttweaker.command.info.entity.type.bracket" : "Tipo di entità: %s",
+  "crafttweaker.command.list.check.log" : "L'elenco %s è stato generato! Puoi visualizzarlo nel file %s.",
+  "crafttweaker.command.misc.block" : "Blocco",
+  "crafttweaker.command.misc.blockstate" : "Proprietà del blocco",
+  "crafttweaker.command.misc.ctgui" : "L'interfaccia di CraftTweaker era limitante e piena di bug e per questo è stata rimossa. Ti consigliamo di imparare ZenScript instead - puoi consultare la wiki (/ct wiki) o chiedere aiuto nel server Discord (/ct discord).",
```

Comment:
Random "instead" left in the translated string

Reply:
Should be fixed.